src/Processors/Formats/Impl/Parquet/ParquetLeafColReader.cpp

```diff
@@ -639,19 +654,23 @@ std::unique_ptr<ParquetDataValuesReader> ParquetLeafColReader<TColumn>::createDi
 }
 
 
-template class ParquetLeafColReader<ColumnUInt8>;
-template class ParquetLeafColReader<ColumnInt32>;
-template class ParquetLeafColReader<ColumnUInt32>;
-template class ParquetLeafColReader<ColumnInt64>;
-template class ParquetLeafColReader<ColumnUInt64>;
-template class ParquetLeafColReader<ColumnBFloat16>;
```

Comment:
@alexey-milovidov did you add it just for the sake of adding it? As far as I can tell, this line has no effect since [makeLeafReader](https://github.com/ClickHouse/ClickHouse/blob/3ae055c74e7ce24100d3fa2348258408af8a5d1d/src/Processors/Formats/Impl/Parquet/ParquetRecordReader.cpp#L97) is [never called](https://github.com/ClickHouse/ClickHouse/blob/3ae055c74e7ce24100d3fa2348258408af8a5d1d/src/Processors/Formats/Impl/Parquet/ParquetRecordReader.cpp#L257) with `ColumnBFloat16`.

Reply:
I have no idea about this code.

Reply:
Git blame on https://github.com/ClickHouse/ClickHouse/blame/master/src/Processors/Formats/Impl/Parquet/ParquetLeafColReader.cpp says you authored it